src/Pos/Launcher/Runner.hs

```diff
@@ -173,6 +173,7 @@ runRawRealMode res np@NodeParams {..} sscnp listeners outSpecs (ActionSpec actio
   where
     LoggingParams {..} = bpLoggingParams npBaseParams
 
+-- TODO: remove this function?
```

Comment:
System start time is still necessary. Formerly in development mode nodes obtained it by having one time-lord and multiple time-slaves. So we have some workers, listeners, messages. Now we want it to be passed via CLI and don't want to have old machinery.